hw/ip/rstmgr/data/rstmgr.hjson.tpl

```diff
@@ -135,47 +135,70 @@
       name:    "por_n",
       act:     "rcv",
       width:   "${len(power_domains)}"
+      desc:    '''
+        Root power on reset signals from ast.
+        There is one root reset siganl for each core power domain.
```

Comment:
nit: siganl

Reply:
ugh, sorry for the typo, not sure why my spell check didn't flag this...